### PR TITLE
feat: add publish mode parameter

### PR DESCRIPTION
**Pull request by scepter914**
Add publish mode parameter `sequential_publish`.

So far, this radar driver publishes the outputs every cycle it receives from can topic.
This specification leads to make output topic very high frame rate (> 500 hz) and it takes rosbag storage up and lack the output data.
So this PR adds parameter `sequential_publish` and if `sequential_publish` is false it makes it stable and suppress the frame rate.

### Test

- Before
  - The driver successively publish RadarTrack.
  - Instead the driver sometime loss the object data.
  - When `sequential_publish` parameter is true, the driver behave this way.

```
$ ros2 topic hz /sensing/radar/front_center/objects_raw

average rate: 551.306
	min: 0.000s max: 0.003s std dev: 0.00104s window: 553

$ ros2 topic echo /sensing/radar/front_center/objects_raw --no-arr | grep length

tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 69>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 69>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 69>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 69>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 14>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 41>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 74>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 3>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 35>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 4>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 43>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 77>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 77>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 77>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 77>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 0>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 30>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 80>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 80>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 80>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 6>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 38>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 76>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 76>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 76>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 76>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 76>'
```

- After
  - The driver publish RadarTrack by trigger from the device.
  - This change leads to low frame rate but make the driver stable.
  - When `sequential_publish` parameter is false, the driver behave this way.

```
$ ros2 topic hz /sensing/radar/front_center/objects_raw
average rate: 12.424
	min: 0.072s max: 0.152s std dev: 0.01979s window: 14

$ ros2 topic echo /sensing/radar/front_center/objects_raw --no-arr | grep length

tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 74>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 60>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 53>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 64>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 68>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 66>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 48>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 57>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 70>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 61>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 55>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 62>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 69>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 75>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 60>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 58>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 68>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 66>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 86>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 94>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 87>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 54>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 59>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 78>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 59>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 65>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 84>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 85>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 38>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 53>'
tracks: '<sequence type: radar_msgs/msg/RadarTrack, length: 88>'
```

Signed-off-by: scepter914 scepter914@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,15 +55,21 @@ ros2 launch continental_ars408_socket_can.launch.xml
...
-
-### continental_ars408_socket_can.xml
-
-- The launch file will initiate two nodes:
-  1. socketcan_bridge to read from `can0` and publish the CAN msg in `can_raw`
-  1. Continental ARS408 driver will read the `can_raw`, parse and publish `RadarTrack` or `RadarReturn`
+- `output_frame`
+  - The string parameter of the output frame id
+- `sequential_publish`
+  - The bool parameter to publish outputs sequentially.
```
Comment (drwnz) [inline README.md:61]: Suggest:
`- The bool parameter to determine output publishing behaviour.`

File: README.md [hunk 2]
```diff
@@ -55,15 +55,21 @@ ros2 launch continental_ars408_socket_can.launch.xml
...
+  - If this parameter is false (default parameter), the driver publish output after receiving all 1 cycle sequential data from can data topic.
```
Comment (drwnz) [inline README.md:62]: Suggest:
- If this parameter is set to `false` (default value), the driver will publish output after receiving a complete cycle of sequential data from the CAN data topic.

File: README.md [hunk 3]
```diff
@@ -55,15 +55,21 @@ ros2 launch continental_ars408_socket_can.launch.xml
...
+  - If this parameter is true, the driver publish output  every time receiving from can data topic.
```
Comment (drwnz) [inline README.md:63]: Suggest:
- If this parameter is set to `true`, the driver will publish output every time data is received from the CAN data topic.

**Review by 0x126 — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (scepter914) [reply README.md]: fixed 22e4591

File: README.md (hunk 2 above)
Comment (scepter914) [reply README.md]: fixed 22e4591

**Review by drwnz — APPROVED** [review]
LGTM

**Pull request merged**